go/protocol/chat1/extras.go

```diff
@@ -3157,3 +3157,26 @@ func (r EmojiLoadSource) IsHTTPSrv() bool {
 	}
 	return typ == EmojiLoadSourceTyp_HTTPSRV
 }
+
+func TeamToChatMemberDetails(teamMembers []keybase1.TeamMemberDetails) (chatMembers []ChatMemberDetails) {
+	chatMembers = make([]ChatMemberDetails, len(teamMembers))
+	for i, teamMember := range teamMembers {
+		chatMembers[i] = ChatMemberDetails{
+			Uv:       teamMember.Uv,
```

Comment:
can you drop the UV and just have uid? chat doesn't need to know about eldest seqno